.github/CONTRIBUTING.md

```diff
@@ -61,7 +61,8 @@ Here's a full example:
 $ ./script/glide.sh get github.com/foo/bar
 # install another dependency, this time for the integration tests
 $ ( cd integration && ../script/glide.sh get github.com/baz/quuz )
-# generate (Only required to integrate other components such as web dashboard)
+# generate
+# (Only required to integrate other components such as web dashboard and if you change something there — web, templates)
```

Comment:
>if you change something there [being the dashboard]

The templates aren't really related to the dashboard, are they?

How about rephrasing it like this:

```
# generate
# (required to merge non-code components into the final binary, such as the web dashboard and provider Go templates)
```